src/login/containers/login/login.component.ts

```diff
@@ -10,10 +12,15 @@ import {Component, OnInit} from '@angular/core';
 })
 export class LoginComponent implements OnInit {
 
-  constructor() {}
+  constructor( private store: Store<fromStore.LoginState>) {}
 
   ngOnInit(): void {
 
   }
 
+  onSignIn(f, event): void {
+    event.preventDefault();
```

Comment:
i really do not like this... peventDefault that is so yesterday

Reply:
removed